### DISCUSSION TITLE
Gift card on cart page

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#228] [1. Help]
Hi @vovayatsyuk 

On the cart page with Breeze applied gift card is not showing in the total section.
and the "Check gift card status and balance" button is also not working, works fine when I exclude the cart from Breeze, so could you please check? has this been developed in Breeze?

![image](https://github.com/breezefront/community/assets/19511407/fd27420b-8f56-43d7-8747-528c9ab0e302)

**Comment by vovayatsyuk** [comment]
Is this a Magento Commerce Gift Cards feature? No, we didn't integrate it.

**Reply by pk-pankaj** [reply]
Yes, this is from Commerce.

There is also a gift wrapping feature that does not integrate with the Breeze right?

![image](https://github.com/breezefront/community/assets/19511407/fd625067-139b-4c93-a9d6-446602393d17)

![image](https://github.com/breezefront/community/assets/19511407/a07fb762-f6ca-4045-ad5f-92673903dea8)

**Reply by vovayatsyuk** [reply]
Gift options, gift wrapping - not integrated.
Gift message - integrated.

**Reply by pk-pankaj** [reply]
So any plan to integrate these in the next release, otherwise we can exclude the cart from Breeze to get these work.

**Reply by vovayatsyuk** [reply]
Unfortunately, not in the next one. I think we will create a separate package for the commerce integrations in a few months.
